<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ParamEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QListWidget" name="paramListWidget" >
         <property name="dragEnabled" >
          <bool>true</bool>
         </property>
         <property name="dragDropMode" >
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2" >
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="pathLineEdit" >
            <property name="minimumSize" >
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Atoms</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="atomSpinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_22" >
            <property name="text" >
             <string>Atom</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QListWidget" name="atomListWidget" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="maximumSize" >
             <size>
              <width>140</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QComboBox" name="typeComboBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <item>
             <property name="text" >
              <string>float</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>int</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>str</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>bool</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Bang</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="defaultEditLabel" >
            <property name="text" >
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="defaultEdit" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" >
           <widget class="QLabel" name="defaultSpinBoxLabel" >
            <property name="text" >
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" >
           <widget class="QDoubleSpinBox" name="defaultSpinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="maximum" >
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" >
           <widget class="QDoubleSpinBox" name="minSpinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <double>-999999.000000000000000</double>
            </property>
            <property name="maximum" >
             <double>999999.000000000000000</double>
            </property>
            <property name="value" >
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" >
           <widget class="QDoubleSpinBox" name="maxSpinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum" >
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="9" column="0" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>WIdget</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" >
           <widget class="MyLineEdit" name="widgetEdit" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="frame" >
             <bool>false</bool>
            </property>
            <property name="readOnly" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>WIdget slot</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1" >
           <widget class="QSpinBox" name="widgetSlot" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="frame" >
             <bool>true</bool>
            </property>
            <property name="readOnly" >
             <bool>false</bool>
            </property>
            <property name="buttonSymbols" >
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
           </widget>
          </item>
          <item row="12" column="1" >
           <widget class="QPushButton" name="newButton" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>New</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1" >
           <widget class="QPushButton" name="deleteButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>283</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel" >
     <x>187</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
